src/main/java/src/main/java/duke/Event.java

```diff
@@ -0,0 +1,38 @@
+package src.main.java.duke;
+
+/**
+ * Class to represent an event as a task
+ */
+
```

Comment:
Should there be spaces between documentation block and the method/class?
I believe the following is mentioned in the coding standard

> Punctuation behind each parameter description -No blank line between the documentation block and the method/class

I noticed the same issue in multiple places